### DISCUSSION TITLE
RTL DetailsList

### DISCUSSION THREAD

**Discussion by avsrot** [microsoft/powercat-creator-kit#244] [Q&A]
For Hebrew and Arabic makers -
Is it possible to have the DetailsList in an RTL fashion? (Say, with an "Align" property)
I mean - have the scroll bar on the left and columns go right to left with selection radio buttons on the right.

**Comment by yaseralhosani** [comment]
Hi @avsrot,

I have already requested text alignment. I hope this applies to all components (wherever possible) for the next enhancements.
[#223](https://github.com/microsoft/powercat-creator-kit/issues/223)
